mula/scheduler/context/context.py

```diff
@@ -85,6 +84,39 @@ def __init__(self) -> None:
         self.stop_event: threading.Event = threading.Event()
 
         # Repositories
+        if not self.config.database_dsn.startswith("postgresql"):
+            raise Exception("PostgreSQL is the only supported database backend")
+
         datastore = sqlalchemy.SQLAlchemy(self.config.database_dsn)
         self.task_store: stores.TaskStorer = sqlalchemy.TaskStore(datastore)
         self.pq_store: stores.PriorityQueueStorer = sqlalchemy.PriorityQueueStore(datastore)
+
+        # Metrics collector registry
+        self.metrics_registry: CollectorRegistry = CollectorRegistry()
+
+        Info(
+            name="app_settings",
+            documentation="Scheduler configuration settings",
+            registry=self.metrics_registry,
+        ).info(
+            {
+                "monitor_organisations_interval": str(self.config.monitor_organisations_interval),
```

Comment:
These should also be separate metrics instead of labels.

Reply:
Is that also true for values of configuration settings? I guess this should probably be known by the person who configured the scheduler., and should not be present at all in a metrics endpoint. But thought of these settings as: `"version": "0.0.1"` etc. hence I opted for `Info`.

I referenced the example of

```python
from prometheus_client import Info
i = Info('my_build_version', 'Description of info')
i.info({'version': '1.2.3', 'buildhost': 'foo@bar'})
```

Reply:
I see! Since these are config values they shouldn't change often so that should be fine. Apparently the [Prometheus docs also include `foobar_build_info` as an example](https://prometheus.io/docs/practices/naming/).